### ISSUE TITLE
new derivative package seems in error

### ISSUE THREAD

**Issue by dgreen** [amaxwell/tlutility#76]
Not sure this is the right place to report but...

The new derivative package being pushed out includes dramatist.pdf which is another package.

**Comment by amaxwell** [comment]
Yes, this is definitely the place to report that, and I'll try and figure out what's going on. Thanks!

**Comment by amaxwell** [comment]
It looks like this is `texdoc` returning garbage when a package isn't installed.
>$ texdoc --list --nointeract --machine derivative
>derivative	7.0	/usr/local/texlive/2019/texmf-dist/doc/latex/dramatist/dramatist.pdf		>Package documentation
>derivative	1.8	/usr/local/texlive/2019/texmf-dist/doc/latex/dramatist/README		Package >Readme
